src/index.ts

```diff
@@ -1,4 +1,4 @@
 export { defineBddConfig, BDDInputConfig } from './config';
 export { createBdd } from './stepDefinitions/createBdd';
-export { test } from './run/bddFixtures';
+export { test, BddFixtures, bddFixtures } from './run/bddFixtures';
```

Comment:
I have some thoughts about adding worker-scoped bdd fixtures type, e.g. `BddFixturesWorker`. 
In that case on the user-land we will need to have the following call to get things work correctly:
```ts
import { test as ctBase } from '@playwright/experimental-ct-react';
import { BddFixtures, BddFixturesWorker, bddFixtures } from 'playwright-bdd';

const bddCtBase = ctBase.extend<BddFixtures, BddFixturesWorker>(bddFixtures);
export const test = bddCtBase.extend<Fixtures>({ ... });
```
Potential problem here is that it's easy to forget about `BddFixturesWorker`.
My suggestion is to export single function `createBddTest(baseTest)` that will internally call `test.extend` with all needed bdd fixtures and types:
```ts
import { test as ctBase } from '@playwright/experimental-ct-react';
import { createBddTest } from 'playwright-bdd';

export const test = createBddTest(ctBase).extend<Fixtures>({ ... });
```

Reply:
Good reasoning, I'll update this!

Reply:
done